cms/templates/cms/home_page.html

```diff
@@ -215,19 +210,15 @@ <h2 class="quote-text">"Inverted admissions has the potential to disrupt traditi
         <div class="modal-body">
           <h2>MicroMasters is a new credential</h2>
           <p>
-          MIT has created a new credential for online learners: the “MicroMasters,” which will be granted by MITx
-          (MIT’s online learning initiative) to students who do exceptionally well in a given set of graduate-level
-          online courses and do well in a subsequent exam. The credential will first be available to students who register for the SCM program’s online courses.
+          MIT has created a new credential for online learners: the “MicroMasters,” which will be granted by MITx (MIT’s online learning initiative) to students who do exceptionally well in a given set of graduate-level online courses, and in subsequent exam(s).
```

Comment:
We're using curly quotes here, instead of straight quotes -- we should probably be consistent.